zephyr/scripts/partitions/esp_genpartition.py

```diff
@@ -6,6 +6,7 @@
 from datetime import datetime
 
 output_path = ""
+start_year = 2024
```

Comment:
2025?

Reply:
common partition tables were introduced last year, therefore 2024.
The proper file creation date should be fetched from the existing file(s), but I didn't want to overcomplicate things.